config/site_settings.yml

```diff
@@ -2437,6 +2437,11 @@ developer:
     list_type: compact
     allow_any: false
     refresh: true
+  experimental_redesigned_about_page_groups:
+    default: ""
+    type: group_list
+    client: true
```

Comment:
Let's keep this hidden, I think in this case it's a setting that doesn't really need to be trialled widely / be opt-in. We can just turn it on for everyone when we are happy.

Also you won't need `client: true` once you make the current user serializer change I mentioned